src/liballoc/arc.rs

```diff
@@ -418,7 +418,7 @@ impl<T: Sync + Send> Clone for Weak<T> {
 }
 
 #[unsafe_destructor]
-#[experimental = "Weak pointers may not belong in this module."]
+#[stable]
```

Comment:
Don't read anything into the `#[stable]` here other than: *if* we have `Weak`, we'll keep the `Drop` impl.